tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -406,6 +406,18 @@
       "3.*.*"
     ]
   },
+  {
+    "IntegrationName": "MSTest2",
+    "SampleProjectName": "Samples.MSTestTests2",
+    "NugetPackageSearchName": "MSTest.TestFramework",
+    "MinVersion": "2.0.0",
+    "MaxVersionExclusive": "4.0.0",
+    "SpecificVersions": [
+      "2.0.*",
+      "2.*.*",
+      "3.*.*"
+    ]
+  },
```

Comment:
Isn't this the same as above? If so, I don't think we need to add it + all the associated other package version changes?